codec-http/src/main/java/io/netty/handler/codec/http/cookie/CookieDecoder.java

```diff
@@ -81,4 +91,8 @@ protected DefaultCookie initCookie(String header, int nameBegin, int nameEnd, in
         cookie.setWrap(wrap);
         return cookie;
     }
+
+    public interface CookieErrorHandler {
```

Comment:
As discussed in https://github.com/netty/netty/pull/6744/#discussion_r116452809 I think adding a `STRICT` and `LAX` options should be good enough and we can make this interface an implementation detail (not expose it publicly).